README.adoc

```diff
@@ -36,7 +36,7 @@ $ gitbook install
 $ gitbook serve -w
 ----
 
-Once above commands executes successfully, the http format document can be test locally via `http://localhost:4000/`.
+Once above commands execute successfully (may take a few minutes), you should see the "Serving book at ..." message and the http format document can be tested locally via `http://localhost:4000/`.
```

Comment:
I've added that, because at least on my pc gitbook didn't output anything when building for like a minute or so. I thought it wasn't working, but all I needed was patience...